test/Microsoft.Azure.WebJobs.Host.EndToEndTests/AsyncChainEndToEndTests.cs

```diff
@@ -175,14 +175,6 @@ public AsyncChainEndToEndTests(TestFixture fixture)
                 Assert.True(queueProcessorFactory.CustomQueueProcessors.All(p => p.Context.Queue.Name.StartsWith("asynce2eq")));
                 Assert.True(queueProcessorFactory.CustomQueueProcessors.Sum(p => p.BeginProcessingCount) >= 2);
                 Assert.True(queueProcessorFactory.CustomQueueProcessors.Sum(p => p.CompleteProcessingCount) >= 2);
-
-                Assert.Equal(19, storageClientFactory.TotalBlobClientCount);
```

Comment:
How did these change? A bit quirky, but I had these in place I believe to verify custom factories were being called?

Reply:
These were very implementation-specific and the implementation is changed.  They weren't really testing something meaningful to an end-user. 

---
In reply to: [151528273](https://github.com/Azure/azure-webjobs-sdk/pull/1422#discussion_r151528273) [](ancestors = 151528273)